consensus/state_sync/module.go

```diff
@@ -34,9 +25,14 @@ type StateSyncModule interface {
 	HandleGetBlockResponse(*typesCons.GetBlockResponse) error
 
 	IsServerModEnabled() bool
-	EnableServerMode()
+	EnableServerMode() error
+	DisableServerMode() error
 
 	SendStateSyncMessage(msg *typesCons.StateSyncMessage, nodeAddress cryptoPocket.Address, height uint64) error
+	IsOutOfSync() bool
```

Comment:
s/IsOutOfSync/IsSynched

Note that it's the opposite (so you need to change logic) but it's clearer.